catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -118,7 +119,7 @@
      */
     protected static final String UPDATE_QUERY_MODE = "update";
 
-    private static final int DEFAULT_MAX_START_INDEX = 50000;
+    static final int DEFAULT_MAX_START_INDEX = 50000;
```

Comment:
✏️  We could add a comment here saying that this is package private for unit testing purposes.